package.json

```diff
@@ -16,6 +16,7 @@
     "webdriver": "webdriver-manager update && webdriver-manager start"
   },
   "devDependencies": {
+    "adm-zip": "^0.4.11",
```

Comment:
I think this should be a prod dependency (not a dev dependency) and it should be in the `api/package.json` not in this top level one.